examples/atari/a3c/README.md

```diff
@@ -0,0 +1,116 @@
+# A3C
+This example trains an Asynchronous Advantage Actor Critic (A3C) agent, from the following paper: [Asynchronous Methods for Deep Reinforcement Learning](https://arxiv.org/abs/1602.01783). 
+
+## Requirements
+
+- atari_py>=0.1.1
+- opencv-python
+
+## Running the Example
+
+```
+python train_a3c.py [options]
+```
+
+### Useful Options
+- `--gpu`. Specifies the GPU. If you do not have a GPU on your machine, run the example with the option `--gpu -1`. E.g. `python train_dqn.py --gpu -1`.
+- `--env`. Specifies the environment. 
+- `--render`. Add this option to render the states in a GUI window.
+- `--seed`. This option specifies the random seed used.
+- `--outdir` This option specifies the output directory to which the results are written.
+
+To view the full list of options, either view the code or run the example with the `--help` option.
+
+## Results
+These results reflect ChainerRL  `v0.6.0`. The ChainerRL score currently consists of a single run. The reported results are compared against the scores from the [Noisy Networks Paper](https://arxiv.org/pdf/1706.10295.pdf), since the original paper does not report scores for the no-op evaluation protocol.
+
+We use the best intermediate scores on each domain to evaluate A3C.
```

Comment:
Did they use the best intermediate scores? Can you add source of it?

Reply:
https://github.com/chainer/chainerrl/pull/433#issuecomment-493937532

It's not clear, I emailed the authors, but no response. What do you think of this: 

- The noisy networks paper uses an intermediate evaluation, not the final network evaluation. Since the noisy nets paper has the results we're comparing against, I use that evaluation protocol.
- However, the evaluation protocol isn't 100% clear in the noisy nets paper: "Per-game
maximum scores are computed by taking the maximum raw scores of the agent and then averaging
over three seeds. However, for computing the human normalised scores in Figure 2, the raw scores
are evaluated every 1M frames and averaged over three seeds." Given that they imply that there's something different between the first and second sentence, I'm confused. Because it seems to me that in both evaluations, they evaluate every 1M frames and take the maximum performance.

Reply:
As the paper says "for computing the human normalised scores in Figure 2", the second sentence is only about Figure 2, which plots scores throughout training, not providing a per-game scalar score. I think this explains the difference in the first and second sentences.

Anyway, can you add explanation so readers can see why you used the best intermediate scores? Such explanation is important especially when it's not 100% clear how they did it.

Reply:
Confirmed with the author. What I wrote in the README matches what they did in the noisy networks paper.